project-api/project-api-test/src/test/resources/myproject_with_lang_import/modules/storage/db.bal

```diff
@@ -0,0 +1,2 @@
+public function initDatabase() {
+}
```

Comment:
We need to add new line on these places. This red circle with - sign indicates that

Reply:
added in commit 274babad95104c166f871220329483954e88cf7b